src/middleware/AuthNavbarMiddleware.ts

```diff
@@ -0,0 +1,15 @@
+import express from "express";
```

Comment:
Change file name

Reply:
This has been corrected and committed.

Reply:
You still need to change the actual file name

Reply:
My mistake, I thought the comment was for the method name, not the actual file name. Regardless, the file name has now been modified and committed.